libshvchainpack/src/chainpack/rpcmessage.cpp

```diff
@@ -140,7 +140,7 @@ RpcFrame RpcFrame::fromFrameData(const std::string &frame_data)
 		auto pos = in.tellg();
 		if(pos < 0)
 			throw ParseException(CCPCP_RC_MALFORMED_INPUT, "Metadata missing", -1, {});
-		auto data = std::string(frame_data, pos);
+		auto data = std::string(frame_data, static_cast<size_t>(pos));
```

Comment:
I'm afraid of useless cast warning on platforms where `tellg` return `size_t`. I also don't know how to handle this situation.